logs_sdk/test/opentelemetry/sdk/logs/export/batch_log_record_processor_test.rb

```diff
@@ -204,6 +204,10 @@ def to_log_record_data
     end
 
     it 'logs a warning if a log record was emitted after the buffer is full' do
+      # This will be fixed as part of Issue #1701
+      # https://github.com/open-telemetry/opentelemetry-ruby/issues/1701
+      skip if RUBY_ENGINE == 'jruby'
+
```

Comment:
@mwear  - Until I can put in a real fix, I think we should skip this test for JRuby. Does that sound good to you?